src/components/GoodsList/GoodsList.tsx

```diff
@@ -0,0 +1,42 @@
+import { FC } from 'react';
+import { SortType } from '../../types/SortType';
+
+type Props = {
+  goods: string[],
+  sortType: SortType,
+  isReversed: boolean,
+};
+
+export const GoodsList: FC<Props> = ({ goods, sortType, isReversed }) => {
+  const renderedGoods = [...goods];
+
+  switch (sortType) {
+    case SortType.ALPHABET:
+      renderedGoods.sort((a, b) => a.localeCompare(b));
+      break;
+
+    case SortType.LENGTH:
+      renderedGoods.sort((a, b) => a.length - b.length);
+      break;
+
+    default:
+      // do nothing
+  }
+
+  if (isReversed) {
+    renderedGoods.reverse();
+  }
```

Comment:
not important, but will be great to create separate function for list transforming, that will accept needed parameters and will return the refactored list